src/LuaPiGui.cpp

```diff
@@ -423,11 +423,30 @@ static int l_pigui_begin(lua_State *l)
 	return 1;
 }
 
+/*
```

Comment:
You should probably add an `/* Interface: ui */` block here at the start of the documentation for pass-through functions, as otherwise all of these functions will be grouped under the `PiGui` interface.

Reply:
I tried with both the above suggested, and 
```
/* 
 * Interface: ui
 */
```

But doesn't insert any such section.

Reply:
More NaturalDocs v2 functionality then, I suppose.